modules/arrow/src/lib/constants.js

```diff
@@ -0,0 +1,4 @@
+export const VECTOR_TYPES = {
```

Comment:
Do we need one more set of meta types? Can we use arrow types?

Reply:
> Do we need one more set of meta types?

There is a specific list of vector types in Arrow. We can extend other types support in separate PRs.

> Can we use arrow types?

This is an interface design question. If encoder should consume already initialized Arrow's vectors, then "VECTOR_TYPES" is useless here. Otherwise we should pass into the encoder a type of specific array to create a vector of corresponding type.